src/Workspaces/Core/Portable/Shared/TestHooks/AsynchronousOperationListener.cs

```diff
@@ -72,6 +78,8 @@ private void Decrement_NoLock(AsyncToken token)
                 }
 
                 _pendingTasks.Clear();
+                _blockedOnCompletionTokenSource = new CancellationTokenSource();
+                _blockedOnCompletionToken = _blockedOnCompletionTokenSource.Token;
```

Comment:
Is there ever a case where _blockedOnCompletionToken isn't just equal to _blockedOnCompletionTokenSource.Token? My assumption is if there are two fields there some subtlety why it's necessary...

Reply:
➡️ Now only one field